### PR TITLE
Add camel-bean-validator wrapper

### PR DESCRIPTION
**Pull request by ozgurkisir-qlik**
Add Camel bean validator wrapper

### CODE REVIEW

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -522,16 +522,15 @@
...
-<!--        <bundle>mvn:org.jboss.logging/jboss-logging/${jboss-logging-version}</bundle>-->
-<!--        <bundle>mvn:com.fasterxml/classmate/${classmate-version}</bundle>-->
-<!--        <bundle>mvn:org.glassfish/jakarta.el/${jakarta-el-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel.karaf/camel-bean-validator-osgi/${project.version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-bean-validator/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-bean-validator' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <bundle>wrap:mvn:org.hibernate.validator/hibernate-validator/${hibernate-validator-version}</bundle>
+        <bundle>wrap:mvn:jakarta.validation/jakarta.validation-api/3.0.2</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: Please use the `jakarta-validation` feature instead

File: features/src/main/feature/camel-features.xml [hunk 2]
```diff
@@ -522,16 +522,15 @@
...
-<!--        <bundle>mvn:jakarta.validation/jakarta.validation-api/${jakarta-validation-api-version}</bundle>-->
-<!--        <bundle>mvn:org.jboss.logging/jboss-logging/${jboss-logging-version}</bundle>-->
-<!--        <bundle>mvn:com.fasterxml/classmate/${classmate-version}</bundle>-->
-<!--        <bundle>mvn:org.glassfish/jakarta.el/${jakarta-el-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel.karaf/camel-bean-validator-osgi/${project.version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-bean-validator/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-bean-validator' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <bundle>wrap:mvn:org.hibernate.validator/hibernate-validator/${hibernate-validator-version}</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: This is already an OSGI bundle, no need to wrap it

Comment (ozgurkisir-qlik) [reply features/src/main/feature/camel-features.xml]: Done

File: features/src/main/feature/camel-features.xml (hunk 1 above)
Comment (ozgurkisir-qlik) [reply features/src/main/feature/camel-features.xml]: Done

File: features/src/main/feature/camel-features.xml [hunk 3]
```diff
@@ -522,16 +522,15 @@
...
-<!--        <bundle>mvn:org.glassfish/jakarta.el/${jakarta-el-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel.karaf/camel-bean-validator-osgi/${project.version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-bean-validator/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-bean-validator' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature>jakarta-validation</feature>
+        <bundle>mvn:org.hibernate.validator/hibernate-validator/${hibernate-validator-version}</bundle>
+        <bundle>wrap:mvn:org.jboss.logging/jboss-logging/${jboss-logging-version}</bundle>
+        <bundle>wrap:mvn:com.fasterxml/classmate/1.5.1</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: We don't need the wrap protocol as it is already an OSGI bundle

File: features/src/main/feature/camel-features.xml [hunk 4]
```diff
@@ -522,16 +522,15 @@
...
+        <bundle>wrap:mvn:jakarta.el/jakarta.el-api/${jakarta-el-api-version}</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: We don't need the wrap protocol as it is already an OSGI bundle

File: features/src/main/feature/camel-features.xml [hunk 5]
```diff
@@ -522,16 +522,15 @@
...
-<!--        <bundle>mvn:com.fasterxml/classmate/${classmate-version}</bundle>-->
-<!--        <bundle>mvn:org.glassfish/jakarta.el/${jakarta-el-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel.karaf/camel-bean-validator-osgi/${project.version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-bean-validator/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-bean-validator' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature>jakarta-validation</feature>
+        <bundle>mvn:org.hibernate.validator/hibernate-validator/${hibernate-validator-version}</bundle>
+        <bundle>wrap:mvn:org.jboss.logging/jboss-logging/${jboss-logging-version}</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: We don't need the wrap protocol as it is already an OSGI bundle

**Review by essobedo — COMMENTED** [review]
Please avoid using the wrap protocol when it is not needed, I explained when to use it, it should be clear now if not please ask

**Review by essobedo — APPROVED** [review]
LGTM

File: features/src/main/feature/camel-features.xml (hunk 3 above)
Comment (ozgurkisir-qlik) [reply features/src/main/feature/camel-features.xml]: Done

File: features/src/main/feature/camel-features.xml (hunk 5 above)
Comment (ozgurkisir-qlik) [reply features/src/main/feature/camel-features.xml]: Done

**Pull request merged**